src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx

```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
+    2. To define the thresholds that open the incident, type the <DNT>**Percentile nth**</DNT> response time value, then select its frequency (above, below, or equal to this value).
+
+       We store the transaction time in milliseconds, although the user interface shows the Critical and Warning values as seconds. If you want to define milliseconds, be sure to include the decimal point in your value.
+  </Collapser>
+
+  <Collapser
+    id="dynamic-targeting-labels"
+    title="Dynamic targeting with labels for apps"
+  >
+    By applying [labels](/docs/data-analysis/user-interface-functions/organize-your-data/labels-categories-organize-apps-servers-monitors#labels) to applications, you can automatically link these [entities](/docs/using-new-relic/welcome-new-relic/get-started/glossary#alert-entity) to your condition. This makes it easy to manage all the applications within a dynamic environment. We recommend using the [agent configuration file](/docs/data-analysis/user-interface-functions/organize-your-data/labels-categories-organize-apps-servers-monitors#config) to best maintain entity labels.
+
+    A single label identifies <DNT>**all**</DNT> entities associated with that label (maximum 10,000 entities). Multiple labels only identify entities which share all the selected labels.
+
+    Using dynamic targeting with your condition also requires that you set an [incident close timer](/docs/alerts-applied-intelligence/new-relic-alerts/alert-incidents/how-alert-condition-incidents-are-closed#time-limit).
+
+    To add, edit, or remove up to ten labels for a condition:
+
+    1. Select <DNT>**APM > Application metric**</DNT> as the product type.
+    2. When identifying entities, select the <DNT>**Labels**</DNT> tab. Search for a label by name, or select a label from the list of categories.
+
+       You can also create conditions directly within the context of what you are monitoring with [Infrastructure](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-alerts-add-edit-or-view-host-alert-information).
+  </Collapser>
+
+  <Collapser
+    id="infrastructure"
+    title="Infrastructure conditions"
+  >
+    You can [create conditions for your resources](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-alerts-add-edit-or-view-host-alert-information) directly from our infrastructure monitoring UI.
+
+    For example, if you want to be notified when we have stopped receiving data from an infrastructure agent, use the [host not reporting](/docs/infrastructure/new-relic-infrastructure/configuration/create-infrastructure-host-not-reporting-condition) condition type. This allows you to dynamically alert on filtered groups of hosts and configure the time window from 5 to 60 minutes.
+  </Collapser>
+</CollapserGroup>
+
+## Apdex and response time conditions [#alert_response]
```

Comment:
Should this be in a `collapsergroup` as well?

Reply:
Thanks!